client/src/leap/soledad/client/sqlcipher.py

```diff
@@ -744,33 +705,6 @@ def _get_syncer(self, url, creds=None):
     # Symmetric encryption of syncing docs
     #
 
-    def _encrypt_syncing_docs(self):
```

Comment:
this method is removed, but it should be removed from the initialization of SQLCipherU1DBSync (where a call to it in the looping call is still made)